Source/Core/InputCommon/ControllerEmu/Setting/NumericSetting.h

```diff
@@ -72,6 +80,7 @@ class NumericSettingBase
 
 protected:
   NumericSettingDetails m_details;
+  std::function<void()> m_callback{nullptr};
```

Comment:
If you initialize this with a lambda that does nothing, like:

```suggestion
  std::function<void()> m_callback = []{};
```

you don't need to check for null within the callback function
```

Reply:
That is good to know. Thanks :>

Reply:
No prob!